### ISSUE TITLE
Remove check for .proto file name to match Descriptor file name

### ISSUE THREAD

**Issue by MexicanAce** [obsidiandynamics/kafdrop#90]
# Problem
Currently, I separate out the large set of `.proto` files to be used (e.g. `customer.proto`, `addressBook.proto`, etc.), but only call `protoc` once during compile time (thus generating a single descriptor file).

```bash
protoc -I=$SRC_DIR --descriptor_set_out=/var/kafka/my_protos.desc $SRC_DIR/protos/*.proto
```

This is not a supported use case in the current [ProtobufMessageDeserializer](https://github.com/obsidiandynamics/kafdrop/blob/master/src/main/java/kafdrop/util/ProtobufMessageDeserializer.java#L61) as we validate the proto filename matches the name of the `.desc` file.

# Suggested Solution
I don't see much benefit in having this validation check, so I'm suggesting we remove it (see PR for exact lines removed).

**Comment by ekoutanov** [comment]
Inviting @ctrongduc and @AndreiIvanitckii for a discussion.

**Comment by ctrongduc** [comment]
I reviewed the code and believe it will fix @MexicanAce's issue, but I wasn't able to test. @AndreiIvanitckii what do you think?

**Comment by btv2018** [comment]
I managed to reproduce the problem. But I am getting another error if I am running the fix from the PR:
```
 A 500 error has occurred: Request processing failed; nested exception is java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
Stack trace
...
kafdrop.util.ProtobufMessageDeserializer.deserializeMessage(ProtobufMessageDeserializer.java:63)
...
```

For me the removed check indeed looks unneeded, but seems it is not enough to fix the problem.
@MexicanAce does the PR fix your problem?

Here is my test example:
The proto definition I use:
```proto
syntax = "proto3";

message Message {
	int64 id = 1;
	string name = 2;
}
```
I compile the description file with `$ protoc --descriptor_set_out=m.desc Message.proto`.

**Comment by AndreiIvanitckii** [comment]
I managed to reproduce the problem. But I am getting another error if I am running the fix from the PR:

```
 A 500 error has occurred: Request processing failed; nested exception is java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
Stack trace
...
kafdrop.util.ProtobufMessageDeserializer.deserializeMessage(ProtobufMessageDeserializer.java:63)
...
```
For me the removed check indeed looks unneeded, but seems it is not enough to fix the problem.
@MexicanAce does the PR fix your problem?

Here is my test example:
The proto definition I use:

```proto
syntax = "proto3";

message Message {
	int64 id = 1;
	string name = 2;
}
```
I compile the description file with `protoc --descriptor_set_out=m.desc Message.proto`.

**Comment by MexicanAce** [comment]
@AndreiIvanitckii I couldn't get the source code to compile locally (my machine was having maven issues), but I figured it was a safe PR to make just looking at the lines I removed... Unsure why you would get an `Index out of bounds` error thrown though.

**Comment by MexicanAce** [comment]
> I managed to reproduce the problem. But I am getting another error if I am running the fix from the PR:
> 
> ```
>  A 500 error has occurred: Request processing failed; nested exception is java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
> Stack trace
> ...
> kafdrop.util.ProtobufMessageDeserializer.deserializeMessage(ProtobufMessageDeserializer.java:63)
> ...
> ```
> 
> For me the removed check indeed looks unneeded, but seems it is not enough to fix the problem.
> @MexicanAce does the PR fix your problem?
> 
> Here is my test example:
> The proto definition I use:
> 
> ```protobuf
> syntax = "proto3";
> 
> message Message {
> 	int64 id = 1;
> 	string name = 2;
> }
> ```
> 
> I compile the description file with `protoc --descriptor_set_out=m.desc Message.proto`.

Within the Kafdrop UI, you would need to select `m.desc` as the Protobuf descriptor and enter `Message` as your protobuf message type name. Does that work?

**Comment by AndreiIvanitckii** [comment]
@MexicanAce the problem is that the code uses the assumption that `.proto` file matches the name of the `.desc` file below. That is why the check is needed.

But the assumption is not correct. I fixed this in #92.
